src/renderer/styles.css

```
@@ -1420,3 +1420,115 @@ body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 200;
+  animation: sudoModalFadeIn 0.2s ease;
+}
+
+.sudo-modal.hidden {
+  display: none;
+}
+
+@keyframes sudoModalFadeIn {
```

Stylelint (17.14.1)

[error] Expected keyframe name "sudoModalFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)